src/replicache-mutation-recovery.test.ts

```diff
@@ -1155,6 +1156,19 @@ test('mutation recovery returns early without running if push is disabled', asyn
   expect(await rep.recoverMutations()).to.equal(false);
 });
 
+test('disableMutationRecovery hidden option', async () => {
+  const rep = await replicacheForTestingNoDefaultURLs(
+    'mutation-recovery-startup',
+    {
+      pullURL: 'https://diff.com/pull',
+      disableMutationRecovery: true,
+    } as unknown as ReplicacheOptions<Record<string, never>>,
```

Comment:
Maybe change `replicacheForTestingNoDefaultURLs` to allow the internal options so the tests do not all have to do this casting dance.